Objects/stringlib/fastsearch.h

```diff
@@ -753,6 +753,22 @@ STRINGLIB(count_char)(const STRINGLIB_CHAR *s, Py_ssize_t n,
 }
 
 
+static inline Py_ssize_t
+STRINGLIB(count_char_no_maximum)(const STRINGLIB_CHAR *s, Py_ssize_t n,
+                                 const STRINGLIB_CHAR p0)
+/* By removing the maximum out of the loop, the compiler can optimize using
+   vectors */
```

Comment:
Please mention explicitly that this function is a specialized version of count_char(). Otherwise, the comment is unclear, "by removing ...": removing compared to what?

Reply:
Whoops, indeed a typical "programmer talking to self" comment. I have made it more clear. Thanks for the suggestion.